### DISCUSSION TITLE
Add a "time" property to the City Objects?

### DISCUSSION THREAD

**Discussion by hugoledoux** [cityjson/specs#125] [Ideas]
we could reuse somehow how they are doing it for the [ogc json-fg](https://github.com/opengeospatial/ogc-feat-geo-json/)

**Comment by balazsdukai** [comment]
Uhm, I have some issues with the "time" property in ogc json-fg (or ogc features api) already. If I understand correctly, the purpose of the "time" property is to facilitate filtering on time.

1. I'm not convinced that every feature has a time property that is actually relevant information.
2. Why is not sufficient to have the time-related data as attributes? Why emphasize it?

For instance, I think the difference is negligible between these two for implementing a filter on "time".

As property:

```json
{
  "CityObjects": {
    "id-1": {
      "time": [...],
      "attributes": {...}
    }
  }
}
```

As attribute:

```json
{
  "CityObjects": {
    "id-1": {
      "attributes": {
        "time": [...]
      }
    }
  }
}
```

But I would not make it a mandatory property in any case.
